### ISSUE TITLE
Issue 3: Train Classifier

### ISSUE THREAD

**Issue by senumiminodya** [senumiminodya/axora-medarchive-solution#3]
- Train Softmax Regression (Logistic Regression with multi_class=multinomial).
- Use cross-validation & hyperparameter tuning.
- Save the trained model.
